### PR TITLE
added a fallback example and added descriptions of the previous examples

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added a attr() fallback example and added descriptions of the other two examples.

### Motivation

Working on the issue [[[CSS] Implement CSS attr(&lt;attr-name&gt;, &lt;attr-fallback&gt;)](https://github.com/mdn/content/issues/29309)](https://github.com/mdn/content/issues/29309)

### Additional details

- [Implement CSS attr(&lt;attr-name&gt;, &lt;attr-fallback&gt;)](https://bugzilla.mozilla.org/show_bug.cgi?id=1448248)

### Related issues and pull requests

- [BCD PR](https://github.com/mdn/browser-compat-data/pull/20899)

### CODE REVIEW

File: files/en-us/web/css/attr/index.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ attr(data-something, "default");
 
 ### content property
 
+This example we are prepending the value of the `data-foo` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("p")}} element.
```
Comment (zfox23) [inline files/en-us/web/css/attr/index.md:153]:
```suggestion
In this example, we prepend the value of the `data-foo` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) to the contents of the {{HTMLElement("p")}} element.
```

File: files/en-us/web/css/attr/index.md [hunk 2]
```diff
@@ -172,6 +174,8 @@ attr(data-something, "default");
 
 {{SeeCompatTable}}
 
+This example we are setting the value of {{CSSXRef("background-color")}} with the value of `data-background` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("div")}} element.
```
Comment (zfox23) [inline files/en-us/web/css/attr/index.md:177]:
```suggestion
In this example, we set the CSS value of {{CSSXRef("background-color")}} to the value of the `data-background` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) assigned to the {{HTMLElement("div")}} element.
```

File: files/en-us/web/css/attr/index.md [hunk 3]
```diff
@@ -203,6 +207,32 @@ attr(data-something, "default");
 
 {{EmbedLiveSample("color_value", "100%", 50)}}
 
+### using fallback
+
+{{SeeCompatTable}}
+
+This example we are appending the value of `data-browser` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) from the {{HTMLElement("p")}} element, if it is missing we _fallback_ to **Unknown**.
```
Comment (zfox23) [inline files/en-us/web/css/attr/index.md:214]:
```suggestion
In this example, we append the value of `data-browser` [`data-*`](/en-US/docs/Web/HTML/Global_attributes/data-*) [global attribute](/en-US/docs/Web/HTML/Global_attributes) to the {{HTMLElement("p")}} element. If the `data-browser` attribute is missing from the {{HTMLElement("p")}} element, we append the _fallback_ value of "**Unknown**".
```

**Review by zfox23 — CHANGES_REQUESTED** [review]
A few suggestions for clarity and grammar. Thank you!

**Comment by dletorey** [comment]
Thank you @zfox23

**Review by zfox23 — APPROVED** [review]

**Pull request merged**